src/main/webapp/js/reducers/field-config-reducer.js

```diff
@@ -27,6 +34,21 @@ export default (state = initialState, action) => {
       return update(state, {
         [entityType]: { $set: _.keyBy(payload.data, 'id') },
       });
+    case START_FETCH_CONFIGS:
+      return {
+        ...state,
+        fetching: true,
+      };
+    case FINISH_FETCH_CONFIGS:
+      return {
+        ...state,
+        fetching: false,
+        fetched: true,
+      };
+    case FETCH_ALL_FIELD_CONFIGS:
```

Comment:
you don't need the FINISH_FETCH_CONFIGS, you can set the 'fetching' and 'fetched' here instead